ipi_tests/examples/README

```diff
@@ -1,6 +1,8 @@
 Automatic testing of examples 
 =============================
 
+Please use 
+
```

Comment:
This seems to be an unfinished thought :)

Reply:
ups, it now deleted.